### PR TITLE
8 point branding

### PR DESCRIPTION
**Pull request by Nikomus**
Some styling that matches the point pages more closely

- https://github.com/wearedarts/games/issues/8

### CODE REVIEW

File: Theme.ts [hunk 1]
```diff
@@ -4,16 +4,19 @@ import type { DefaultTheme } from 'styled-components';
...
-    orange_dark: '#cc5e20',
-    blue: '#425cc7',
-    green: '#095157',
     purple: '#532b3a',
-    grey_dark: '#000000e6',
-    grey_light: '#3c3c3c',
-    black: '#00000033',
+    turquoise: '#57b6b2',
+    turquoise_dark: '#095157',
+    green: '#025157',
```
Comment (katjam) [inline Theme.ts:11]: I went with calling this teal so it matches their brand guide also purple --> plum

File: Theme.ts [hunk 2]
```diff
@@ -140,8 +143,8 @@ export const GlobalStyle = createGlobalStyle`
   //
 
   body {
-    background-color: ${(props) => props.theme.colours.purple};
-    color: ${(props) => props.theme.colours.cream};
+    background-color: ${(props) => props.theme.colours.white};
+    color: ${(props) => props.theme.colours.grey};
```
Comment (katjam) [inline Theme.ts:147]: I've used brand plum as font colour. It seems to work ok. Is this grey what they use on the point site or did you add it?

File: pages/index.tsx [hunk 3]
```diff
@@ -111,16 +126,16 @@ export default function Index() {
...
             <p>How to play information.</p>
             <p>More how to play information.</p>
-          </div>
+          </HowToPlay>
         </Container>
       </Main>
 
       <Footer>
-        <a href='https://wearedarts.org.uk/'>wearedarts.org.uk</a>
+        <a href='https://thepoint.org.uk/'>thepoint.org.uk</a>
```
Comment (katjam) [inline pages/index.tsx:138]: We should note this and remember to ask them which or if they want both.

**Review by katjam — APPROVED** [review]
Couple comments, but I'm happy if you merge like this.

File: Theme.ts (hunk 1 above)
Comment (Nikomus) [reply Theme.ts]: It's quite dark for a teal, but that makes sense.

File: Theme.ts (hunk 2 above)
Comment (Nikomus) [reply Theme.ts:147]: I took it from the website, and cleaned up some of the other colours from there that weren't in use.

**Pull request merged**